tendrl/commons/objects/cluster/atoms/clear_cluster_details/__init__.py

```diff
@@ -0,0 +1,120 @@
+                cluster_tendrl_context.cluster_id
+            )
+            NS._int.client.delete(
+                "/indexes/tags/provisioner/%s" %
+                integration_id
+            )
+            NS._init.client.delete(
+                "/indexes/tags/tendrl/integration/%s" %
+                integration_id
+            )
+
+            nodes = NS._init.client.read(
+                "/clusters/%s/nodes" % integration_id
+            )
+            node_ids = []
+            for node_id in nodes.leaves:
+                node_ids.append(node_id)
+                nc = Node(node_id=node_id).load()
+                node_ips[node_id] = nc.fqdn
+                NS._int.client.delete(
+                    "/nodes/%s" % node_id,
+                    recursive=True
+                )
+                NS._int.client.delete(
+                    "/alerting/nodes/%s" % node_id,
+                    recursive=True
+                )
+                NS._init.client.delete(
+                    "/indexes/tags/tendrl/node_%s" % node_id
+                )
+
+            # Remove the nodes from /indexes/ip
+            idx_ips = NS._int.client.read("/indexes/ip")
+            for node_id_for_ip in idx_ips.leaves:
+                if node_id_for_ip.value in node_ids:
+                    NS._int.client.delete(
+                        "/indexes/ip/%s" % node_id_for_ip.key
+                    )
+
+            # Remove the network details for cluster nodes
+            nws = NS._int.client.read("/networks")
+            for nw in nws.leaves:
+                nodes_nw = NS._int.client.read("/networks/%s" % nw)
+                for node_nw in nodes_nw.leaves:
+                    if node_nw in node_ids:
+                        NS._int.client.delete(
+                            "/networks/%s/%s" % (nw, node_nw),
+                            recursive=True
+                        )
+
+            # Load the gluster servers list and remove the cluster nodes
+            gl_srvr_list = NS._int.client.read(
+                "/indexes/tags/gluster/server"
+            ).value
+            for node_id in node_ids:
+                gl_srvr_list.remove(node_id)
+            NS._init.client.write(
+                "/indexes/tags/gluster/server",
+                gl_srvr_list
+            )
+
+            # Load the tendrl servers list and remove
+            # the cluster nodes
+            tendrl_nodes_list = NS._int.client.read(
+                "/indexes/tags/tendrl/node"
+            ).value
+            for node_id in node_ids:
+                tendrl_nodes_list.remove(node_id)
+            NS._init.client.write(
+                "/indexes/tags/tendrl/node",
+                tendrl_nodes_list
+            )
+
+            # Load the tendrl GI nodes list and remove
+            # the cluster nodes
+            tendrl_gi_nodes_list = NS._int.client.read(
+                "/indexes/tags/tendrl/integration/gluster"
```

Comment:
why is this required

Reply:
If https://github.com/Tendrl/commons/pull/798#discussion_r160599756 is correct then yes this also can be removed.